hole/topological-sort.go

```diff
@@ -73,6 +73,9 @@ func generateTsortTest(length, additional int) string {
 // single-digit numbers
 var _ = answerFunc("topological-sort", func() []Answer {
 	var tests []test
+	staticTest := "8 9\n7 8\n6 7\n5 6\n4 5\n3 4\n2 3\n1 2\n0 1"
+	staticResult := tsort(staticTest)
+	tests = append(tests, test{staticTest, staticResult})
```

Comment:
Note to self, this is fine atm because we currently generate 193 and 194 args per run so with this change it'll be 194 and 194 so no solution will break, in fact some might be shorter to hardcode 194 now, but in general we have to be careful about just slapping new test cases onto an existing stable hole.